storage/grpc_client_test.go

```diff
@@ -23,11 +23,12 @@ import (
 
 	"cloud.google.com/go/storage/internal/apiv2/storagepb"
 	"github.com/google/go-cmp/cmp"
+	"google.golang.org/grpc/mem"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/testing/protocmp"
 )
 
-func TestBytesCodec(t *testing.T) {
+func TestBytesCodecV2(t *testing.T) {
```

Comment:
We should also test with random splits in the encoded response.

Reply:
done, but I also think I'll add some fuzz testing later.

Reply:
Ended up adding splits down to every byte which I think covers this pretty well. Will file a follow up issue to look into fuzzing.